waku.md

```diff
@@ -298,6 +299,21 @@ In order to receive historic messages from a mailserver, a node MUST trust the s
 
 Received envelopes MUST be passed through the Waku envelopes pipelines so that they are picked up by registered filters and passed to subscribers.
 
+### Accounting for resources (experimental)
+
+This is an experimental mode that MAY be implemented. It is heavily inspired by Swarm's SWAP protocol, and works by doing pairwise accounting for resources.
+
+Each node keeps track of resource usage with all other node. Whenever an envelope is received from a node that is expected (fits bloom filter or topic interest, is legal, etc) this is tracked.
+
+Every time epoch (say, every minute or every time an event happens) statistics SHOULD be print of the form:
```

Comment:
printed where?

Reply:
Um, STDOUT or wherever seems appropriate for implementation specific logging? Don't think this needs to be overly prescriptive

Reply:
I don't think whether or not nodes print to STDOUT should be part of the spec, nodes should account which is what we care about. We shouldn't care about UI imo.

Reply:
fair, it isn't meant to be part of the final spec, just experimental placeholder for now while there are no consequences for accounting. We can remove it from the spec and just make sure it happens in clients separately I suppose

Reply:
> Every time epoch (say, every minute or every time an event happens) statistics SHOULD be print of the form: 

=>

>  Every time epoch (say, every minute or every time an event happens) statistics SHOULD be aggregated

Reply:
@oskarth yeah that sounds better.

Reply:
yeah, agree here. As there are other means to collect these statistics such as metrics toolkits.